core/server/master/src/main/java/alluxio/master/file/meta/MutableInode.java

```diff
@@ -162,6 +170,20 @@ public AccessControlList getACL() {
     return mAcl;
   }
 
+  @Override
+  public BitSet getPinnedLocation() { return mPinnedLocation; }
+
+  @Override
+  public List<String> getPinnedLocationList() {
+    List<String> mediaList = ServerConfiguration.getList(PropertyKey.MASTER_TIERED_STORE_GLOBAL_MEDIA, ",");
```

Comment:
Because we are implicitly associating the bit set with media, if the user changes the property (for example removes a tier), all the previously stored state will become incorrect?